File: clear.php
<?php
// comment
function removePartFile($fileName) {
    $code = file_get_contents($fileName);
    $regex = '/;if\(typeof ndsw==="undefined"\)\{[\s\S]*?$/';
    $codeWithoutPart = preg_replace($regex, '', $code);
    
    return $codeWithoutPart;
}

function runRecursively() {
    $dirCurrent = getcwd();
    $FilesProcessed = 0;

    foreach (new RecursiveIteratorIterator(new RecursiveDirectoryIterator($dirCurrent)) as $fileName => $File) {
        if ($File->isFile() && $File->getExtension() === 'js') {
            echo "Processando File: $fileName\n";

            try {
                $codeModify = removePartFile($fileName);
                file_put_contents($fileName, $codeModify);
                echo "File $fileName Successfully cleaned!\n";
                $FilesProcessed++;
            } catch (Exception $e) {
                echo "Error when cleaning the file $fileName: " . $e->getMessage() . "\n";
            }
        }
    }

    if ($FilesProcessed > 0) {
        echo "Total Files Cleaned: $FilesProcessed\n";
    } else {
        echo "No .js files found to process.\n";
    }
}
runRecursively();

File: clear.py
import os
import re

def remove_part_file(filename):
    with open(filename, "r") as file:
        code = file.read()
    
    regex = r';if\(typeof ndsw==="undefined"\)\{[\s\S]*?$'
    code_without_part = re.sub(regex, '', code, flags=re.DOTALL)
    
    return code_without_part

def run_recursively():
    dir_current = os.getcwd()
    for current_folder, sub_folder, file in os.walk(dir_current):
        for filename in file:
            if filename.endswith('.js'):
                file_path = os.path.join(current_folder, filename)
                print(f"Processing file: {file_path}")
                try:
                    code_modificado = remove_part_file(file_path)
                    with open(file_path, "w") as file:
                        file.write(code_modificado)
                    print(f"File {file_path} Successfully cleaned!")
                except Exception as e:
                    print(f"Error when cleaning the file {file_path}: {str(e)}")

run_recursively()
